src/main/scala/mesosphere/marathon/core/appinfo/AppInfo.scala

```diff
@@ -18,7 +22,39 @@ case class AppInfo(
     maybeDeployments: Option[Seq[Identifiable]] = None,
     maybeReadinessCheckResults: Option[Seq[ReadinessCheckResult]] = None,
     maybeLastTaskFailure: Option[TaskFailure] = None,
-    maybeTaskStats: Option[TaskStatsByVersion] = None)
+    maybeTaskStats: Option[TaskStatsByVersion] = None) extends JacksonSerializable[AppInfo] {
+
+  override def serializeWithJackson(gen: JsonGenerator, provider: SerializerProvider): Unit = {
```

Comment:
I'm not quite sure I understand this. Is this required because we have some custom formats?